.gitattributes

```diff
@@ -9,4 +9,5 @@ ffi/diplomat/cpp/docs/** linguist-generated=true
 ffi/diplomat/js/include/** linguist-generated=true
 ffi/diplomat/js/docs/** linguist-generated=true
 *.rs.data linguist-language=Rust
+provider/*/tests/data/**/* linguist-generated=true
```

Comment:
Question/issue: wrong PR for this change?

Reply:
Is that the only blocker?